appium_helpers.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+from appium import webdriver
+from appium.webdriver import errorhandler
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from abc import ABCMeta
+from abc import abstractmethod
+from subprocess import check_output
+from subprocess import CalledProcessError
+from os import getcwd
+from os import environ
+
+
+def get_capabilities(options={}):
+    """
+    Try to use the capabilities defined in ENV variable or
+    then use capabilities defined in 'options' or
+    then use defaults
+    """
+    return {
+        "ipa" : environ.get("APPIUM_APPFILE") or options.get("ipa"),
```

Comment:
Should this be 'app' so we can support both iOS and Android? I think that is how we do on the node implementation.